### PR TITLE
Issue/jsproxy with real vertxbusjs

### PR DESCRIPTION
**Pull request by pmlopes**

### CODE REVIEW

File: src/test/resources/vertx-js/vertx-eventbus.js [hunk 1]
```diff
@@ -0,0 +1,296 @@
+/*
```
Comment (cescoffier) [inline src/test/resources/vertx-js/vertx-eventbus.js]: Shouldn't be better to resolve the file from vertx-web ? (I can do it)

**Comment by vietj** [comment]
looks good to me

File: src/test/resources/vertx-js/vertx-eventbus.js (hunk 1 above)
Comment (pmlopes) [inline src/test/resources/vertx-js/vertx-eventbus.js]: yes it would be better since they would always be in sync

Comment (cescoffier) [inline src/test/resources/vertx-js/vertx-eventbus.js]: ok, give me a minute, I will push on this branch

Comment (cescoffier) [inline src/test/resources/vertx-js/vertx-eventbus.js]: done

**Comment by cescoffier** [comment]
ok for me, I've pushed my modification to keep in sync with the vertx-web client.

**Pull request merged**